db/schema.rb

```diff
@@ -23,6 +23,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "suid"
+    t.integer "school_type", limit: 2
```

Comment:
I'm guessing this PR will have to be rebased once the public/private school one is merged so that this doesn't appear as a new line of code